cmd/manager/main.go

```diff
@@ -123,7 +123,7 @@ func main() {
 	buildHelperAPI := build.NewHelper()
 	nodeAPI := node.NewNode(client)
 	kernelAPI := module.NewKernelMapper(buildHelperAPI, sign.NewSignerHelper())
-	micAPI := mic.NewModuleImagesConfigAPI(client, scheme)
+	micAPI := mic.New(client, scheme)
```

Comment:
I am not sure how I feel about this one. I agree in theory, but in practice is is easier to grep `NewModuleImagesConfigAPI` and we use this convention to all out other interfaces...

Reply:
maybe we need to start changing it. The controller-runtime is doing this also. I think it is easy to find, since we know what is the package